solidus_dev_support.gemspec

```diff
@@ -45,7 +46,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rubocop-rails', '~> 2.3'
   spec.add_dependency 'rubocop-rspec', '~> 2.0'
   spec.add_dependency 'solidus_core', ['>= 2.0', '< 4']
-  spec.add_dependency 'webdrivers', '~> 4.4'
+  spec.add_dependency 'webdrivers', '~> 4.4', '< 6.0'
```

Comment:
If we add `< 6.0`, `~> 4.4` automatically doesn't make sense anymore. I'd propose to just use `> 4.4` so we'll always have the last version available. Or even just remove the version from this dependency.

Reply:
Considering that this gem is used only in development, I'm okay with making the constraint more permissive. I'll update it to `>= 4.4`. Todo.